enterprise/frontend/src/embedding-sdk/lib/default-height.ts

```diff
@@ -0,0 +1,28 @@
+import { CARD_SIZE_DEFAULTS_JSON } from "cljs/metabase.shared.dashboards.constants";
+import type { CardDisplayType } from "metabase-types/api";
+
+/**
+ * How many pixels are in each cell?
+ *
+ * The card size default height are measured in number of cells,
+ * so a display size of 5 cells would be 5 * VIS_CELL_HEIGHT pixels tall.
+ **/
+const VIS_CELL_HEIGHT = 50;
```

Comment:
We could expose this to our theme options as well, but let's do it later when the need arises.

Reply:
nit: we shorten visualization as `viz` not `vis`

```suggestion
const VIZ_CELL_HEIGHT = 50;
```